File: Store.Frontend/src/app/models/orderViewModel.ts
export class OrderViewModel {
    constructor(
        public userId: number,
        public items: string,
        public price: number,
        public statusId: StatusId,
        public mapItemId: number,
        public date: Date,
        public isGuest: boolean,
        public guestId: number
    ) {}
}

export enum StatusId {
    checking = 2,
    accepted = 4,
    received = 8
}

File: Store.Backend/ViewModels/OrderViewModel.cs
using static WebStore.Models.Order;

namespace WebStore.ViewModels
{
    public class OrderViewModel
    {
        public int UserId { get; set; }
        public string Items { get; set; }
        public float Price { get; set; }
        public StatusType StatusId { get; set; }
        public int MapItemId { get; set; }
        public DateTime Date { get; set; }
        public bool IsGuest { get; set; }
        public int GuestID { get; set; }
    }
}
